packages/yoastseo/spec/fullTextTests/testTexts/fr/frenchPaper.html

```diff
@@ -0,0 +1,67 @@
+<div id="mw-content-text" class="mw-content-ltr" dir="ltr" lang="fr">
```

Comment:
`frenchPaper1` threw errors, so instead I copy-pasted back the `frenchPaper2` .js and .html files from trunk.